src/sentry/api/endpoints/project_dynamic_sampling.py

```diff
@@ -243,60 +301,127 @@ def get(self, request: Request, project) -> Response:
         query_time_range: QueryTimeRange = QueryTimeRange(time_now - stats_period, time_now)
 
         try:
-            root_transactions = self.__fetch_randomly_sampled_root_transactions(
+            # Fetch X random trace ids group by count_if(trace.parent_span, equals, "")
+            transactions = self.__fetch_randomly_sampled_transactions(
                 project=project,
                 query=query,
                 sample_size=requested_sample_size,
                 query_time_range=query_time_range,
             )
         except EmptyTransactionDatasetException:
+            # TODO: make response keys in same notation (all camelCase)
```

Comment:
Remove this comment and pull this branch https://github.com/getsentry/sentry/pull/39677 into yours